### PR TITLE
Add icedtea plugin to firefox

### PR DESCRIPTION
**Pull request by pasqui23**

### CODE REVIEW

File: modules/programs/firefox.nix [hunk 1]
```diff
@@ -33,6 +33,11 @@ in
         default = false;
         description = "Whether to enable the unfree Google Talk plugin.";
       };
+      enableIcedtea = mkOption {
```
Comment (rycee) [inline modules/programs/firefox.nix]: Can call this `enableIcedTea` to match the [official capitalization](https://icedtea.classpath.org/wiki/Main_Page).

**Comment by rycee** [comment]
Many thanks for the contribution! In general it looks good, I just added one comment. Once you've fixed that please squash these commits and use "firefox: add icedtea plugin" as commit message as per the [contributing guidelines](https://github.com/rycee/home-manager/blob/master/CONTRIBUTING.md#commits). Then I'd be happy to merge to master.

**Comment by rycee** [comment]
By the way, there is no need to create a new pull request when you add or change commits to your branch. GitHub will update the pull request to match.

**Comment by pasqui23** [comment]
Sorry but I can't seem to understand how to squash commits

**Comment by rycee** [comment]
To squash commits you have to use something called an _interactive rebase_. Here is a good tutorial: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

Once you've squashed the commits locally you have _force push_ them to your branch on GitHub. You can do this with `git push --force-with-lease`.

**Comment by rycee** [comment]
I took the liberty of squashing this. It was rebased to master in b78b2b6b35bc87a032334cb8ef0cbf30fed64ad6. Thanks for the contribution!

**Pull request closed without merging**